\documentclass[12pt, a4paper]{article}
\usepackage[english]{babel}
\usepackage[utf8x]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[a4paper]{geometry}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{graphicx}
\usepackage[colorlinks=true, allcolors=blue]{hyperref}
\usepackage{epsfig,amsfonts}
\usepackage{natbib}
\usepackage{authblk}
\usepackage{subfig}
\usepackage{setspace}
\usepackage{hypcap}
%From: https://tex.stackexchange.com/questions/14364/cross-referencing-between-different-files to connect references from external .tex documents; NOTE: to get xr package to work in Overleaf, followed the directions here https://www.overleaf.com/learn/how-to/Cross_referencing_with_the_xr_package_in_Overleaf ; NOTE: seems like when use {xr} and not {xr-hyper}, things work -- get some 'errors' when use xr-hyper, even though within the document looks like things are working fine?
\usepackage{xr}

\title{Differential complex trait architecture across humans: epistasis identified in non-European populations at multiple genomic scales}
\author[1,2]{Michael C. Turchin}
\author[1,3]{Isabella Ting}
\author[1,4,5,*]{Lorin Crawford}
\author[1,2,*,$\dag$]{Sohini Ramachandran}
\affil[1]{Center for Computational Molecular Biology, Brown University}
\affil[2]{Department of Ecology and Evolutionary Biology, Brown University}
\affil[3]{Department of Computer Science, Brown University}
\affil[4]{Department of Biostatistics, Brown University}
\affil[5]{Center for Statistical Science, Brown University}
\affil[$\ast$]{indicates these authors contributed equally}
\affil[$^\dag$]{To whom correspondence should be addressed: sramachandran@brown.edu}

\begin{document}

\maketitle

\begin{abstract}\label{InterPath-Abstract}
Genome-wide association (GWA) studies have identified thousands of significant genetic associations in humans across a number of complex traits. However, the vast majority of these studies use datasets of predominantly European ancestry \cite{Popejoy2016}. It has generally been thought that complex trait genetic architecture should be transferable across populations of different ancestries, but recent work has shown a number of differences in trait architecture across human ancestries, including heterogeneity in both the identified causal variants and estimated effect sizes 
\cite{Martin2017,Wojcik2019}. Here, we report further evidence that complex trait genetic architecture is fundamentally different among human ancestries by jointly leveraging pathway and epistasis analysis.

Under the assumption that a given complex trait may have differential polygenic architectures across human ancestries, we hypothesize that human populations may also be enriched for differences in epistatic effects. However, since polygenic traits tend to have smaller GWA effect sizes, combining variants via pathway analysis may allow us to better reveal these signals. To accomplish this, we extend the concept of identifying marginal epistasis, moving from testing single variants \cite{Crawford2017} to testing groups of variants for nonlinear association with a trait of interest.

We apply our new method to multiple ancestries present in the UK Biobank \cite{Sudlow2015} and explore multiple pathway-related interaction models. Using morphometric traits we find evidence for genome-wide epistasis in African and other non-European populations. We also find evidence that these trends exists on the SNP and gene levels as well. Results also indicate this may be due to increased heterozygosity in non-European populations. This suggests that non-European populations may be well-suited for identifying non-additive effects in human complex trait architecture; this also suggests further evidence that European populations -- predominantly used for epistasis studies -- may indeed be limited and inaccurate proxies for all human ancestries in complex trait research.
\end{abstract}

\section{Introduction}\label{InterPath-Introduction}

Genome-wide association studies (GWAS) have identified over \textcolor{red}{24,000} significant associations between individual genotypes andcomplex human traits. However, the vast majority of these associations were identified in datasets of primarily European ancestry. In a survey of published GWAS from 2009, it was found that only 4\% of the 1.7 million individuals studied were of non-European ancestry (26). This report gained some brief attention, but an initial increase in the representation of non-European ancestries in GWAS has stagnated since 2014 (27,28); this lack of representation is even more disconcerting given the ongoing explosion of available human genomic data. 

It was largely hoped during the GWAS era that human genetic architecture would be consistent across human ancestries, for instance that causal loci and association effect sizes would remain the same among different human populations (26,29-31). However, recent work has begun to directly interrogate this assumption, and the initial findings are indeed showing that genetic architecture is often not the same between human ancestries; studies have shown that not only are association effect size estimates different between ancestries, but at times even causal loci appear to differ as well (32-35). Additionally, polygenic risk score studies, where published effect size estimates are used to predict phenotypes in other populations, are showing that applying European-based estimates to non-European populations consistently produce erratic and nonsensical results (36-39).

It is clear both that non-European populations are incredibly underrepresented in modern GWAS studies and that European populations cannot act as accurate proxies for the rest of the world. Therefore there is an massive need for work that explores the many aspects of complex trait genetic architecture in non-European populations. To help address these needs, we focus here on investigating the importance of epistasis, or genetic interactions, in complex traits across multiple non-European populations. 

Epistasis, despite being a well-established component of complex trait architecture in multiple model organisms (citations), is still regarded cautiously in human genetics (citations). Recent work has begun to show increased evidence of epistasis playing a role in human trait architecture, but still most of this research has been conducted primarily in datasets of European ancestry. Therefore to address this unexplored area of research, we investigated epistasis at multiple genomic scales across multiple European and non-European populations. We extracted both European and non-European subsets from the UK BioBank (UKB) (citation) and identified evidence for epistasis on the SNP-level, pathway-level, and the genome-wide level. We identify that evidence for epistasis varies across human ancestries, and that African populations often have greater evidence for epistasis. We present a novel framework for combining epistasis and pathway analysis and identify pathways that have both population-specific evidence for epistasis as well as pathways that appear to have significant levels of epistasis worldwide. And lastly, we show beginning evidence that one of the driving factors for the higher levels of epistasis apparent in non-European populations may be greater levels of population-wide genetic diversity.


\section{Results}\label{InterPath-Results}

\subsection{Genome Level Epistasis}\label{InterPath-Results-SNPEpistasis}

To investigate epistasis in multiple human populations, we first extracted a variety of human ancestries from the UKB (see Methods). We collected a total of 8 UKB subsets with an average sample size of XXXX among the non-European populations (Supplementary Table 1), including African, Indian, Chinese, and Pakistani subsets. To maximize our sample sizes per subset, we focused on height and body mass index (BMI) as our complex traits of interest. 

The first approach we took to investigate epistasis was to look at genome-wide estimates of phenotypic PVE. Specifically, we were interested in the following variance component model:

\begin{equation}\label{InterPath-GEMMA-Equation-Model}
 := \sum_{\gamma}w_{\gamma} \gamma
\end{equation}

start with ukbiobank information
GEMMA PVE Results

\subsection{SNP Level Epistasis}\label{InterPath-Results-SNPEpistasis}

PLINK and MAPIT results

\subsection{Pathway Level Epistasis}\label{InterPath-Results-SNPGenomeEpistasis}

Explain motivation for moving into InterPath

\subsubsection{InterPath}

Explain InterPath

\subsubsection{InterPath Results}

Show Results
 - number of hits (barplots)
 - overlap between ancestries / phenotypes
 - which pathways are coming up to the top? modes of epistasis being witnessed/seen?

\subsubsection{Replication in British Subsamples}

\subsection{Relationship Between Epistasis and Genetic Diversity}

\section{Discussion}\label{InterPath-Discussion}


\section{Online Methods}\label{InterPath-Online-Methods}

\subsection{UK BioBank Subsets}

UK BioBank data was applied for and downloaded XXX. We then conducted standard quality control (QC) procedures. Different population subsets were first extracted and grouped using self-identified ancestry. SNPs within each subset were then filtered out using the following criteria: < .01 minor allele frequency (MAF), > .05 genotype missingness, and < 1e-6 hardy-weinberg equilbiribum p-value. Individuals within each subset were then filtered out using the following criteria: 

was then filtered by mino 

Individuals were extracted and grouped bWe conducted standard quality control (QC) procedures

\subsection{PLINK Analyses}

\subsection{MAPIT Analyses}

\subsection{GEMMA Analyses}

\subsection{InterPath Model}

\subsection{InterPath Analyses}

\subsection{Genetic Diversity Analyses}

\section{URLs}\label{InterPath-URLs}

\section{Acknowledgments}\label{InterPath-Acknowledgments}

\section{Author Contributions}\label{InterPath-Author-Contributions}

\section{Competing Interests}\label{InterPath-Competing-Interests}

\begingroup
\bibliographystyle{apalike}
\setstretch{1.0}
%\bibliography{Main}
\endgroup

\clearpage
\section{Supplementary Material}

\captionsetup[figure]{name=Supplementary Figure}
\captionsetup[table]{name=Supplementary Table}
%\setcounter{figure}{0}


\begingroup
\bibliographystyle{apalike}
\setstretch{1.0}
%\bibliography{Supplementary}
\endgroup


\iffalse

\fi

\end{document}
