\documentclass[12pt, a4paper]{article}
\usepackage[english]{babel}
\usepackage[utf8x]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[a4paper]{geometry}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{graphicx}
\usepackage[colorlinks=true, allcolors=blue]{hyperref}
\usepackage{epsfig,amsfonts}
\usepackage{natbib}
\usepackage{authblk}
\usepackage{subfig}
\usepackage{setspace}
\usepackage{hypcap}
%From: https://tex.stackexchange.com/questions/14364/cross-referencing-between-different-files to connect references from external .tex documents; NOTE: to get xr package to work in Overleaf, followed the directions here https://www.overleaf.com/learn/how-to/Cross_referencing_with_the_xr_package_in_Overleaf ; NOTE: seems like when use {xr} and not {xr-hyper}, things work -- get some 'errors' when use xr-hyper, even though within the document looks like things are working fine?
\usepackage{xr}

\title{Differential complex trait architecture across humans: epistasis identified in non-European populations at multiple genomic scales}
\author[1,2]{Michael C. Turchin}
\author[1,3]{Isabella Ting}
\author[1,4,5,*]{Lorin Crawford}
\author[1,2,*,$\dag$]{Sohini Ramachandran}
\affil[1]{Center for Computational Molecular Biology, Brown University}
\affil[2]{Department of Ecology and Evolutionary Biology, Brown University}
\affil[3]{Department of Computer Science, Brown University}
\affil[4]{Department of Biostatistics, Brown University}
\affil[5]{Center for Statistical Science, Brown University}
\affil[$\ast$]{indicates these authors contributed equally}
\affil[$^\dag$]{To whom correspondence should be addressed: sramachandran@brown.edu}

\begin{document}

\maketitle

\begin{abstract}\label{InterPath-Abstract}
Genome-wide association (GWA) studies have identified thousands of significant genetic associations in humans across a number of complex traits. However, the vast majority of these studies use datasets of predominantly European ancestry \cite{Popejoy2016}. It has generally been thought that complex trait genetic architecture should be transferable across populations of different ancestries, but recent work has shown a number of differences in trait architecture across human ancestries, including heterogeneity in both the identified causal variants and estimated effect sizes 
\cite{Martin2017,Wojcik2019}. Here, we report further evidence that complex trait genetic architecture is fundamentally different among human ancestries by jointly leveraging pathway and epistasis analysis.

Under the assumption that a given complex trait may have differential polygenic architectures across human ancestries, we hypothesize that human populations may also be enriched for differences in epistatic effects. However, since polygenic traits tend to have smaller GWA effect sizes, combining variants via pathway analysis may allow us to better reveal these signals. To accomplish this, we extend the concept of identifying marginal epistasis, moving from testing single variants \cite{Crawford2017} to testing groups of variants for nonlinear association with a trait of interest.

We apply our new method to multiple ancestries present in the UK Biobank \cite{Sudlow2015} and explore multiple pathway-related interaction models. Using morphometric traits we find evidence for genome-wide epistasis in African and other non-European populations. We also find evidence that these trends exists on the SNP and gene levels as well. Results also indicate this may be due to increased heterozygosity in non-European populations. This suggests that non-European populations may be well-suited for identifying non-additive effects in human complex trait architecture; this also suggests further evidence that European populations -- predominantly used for epistasis studies -- may indeed be limited and inaccurate proxies for all human ancestries in complex trait research.
\end{abstract}

\section{Introduction}\label{InterPath-Introduction}



\section{Results}\label{InterPath-Results}

\subsection{Genome Level Epistasis}\label{InterPath-Results-SNPEpistasis}

GEMMA PVE Results

\subsection{SNP Level Epistasis}\label{InterPath-Results-SNPEpistasis}

PLINK and MAPIT results

\subsection{Pathway Level Epistasis}\label{InterPath-Results-SNPGenomeEpistasis}

Explain motivation for moving into InterPath

\subsubsection{InterPath}

Explain InterPath

\subsubsection{InterPath Results}

Show Results
 - number of hits (barplots)
 - overlap between ancestries / phenotypes
 - which pathways are coming up to the top? modes of epistasis being witnessed/seen?

\subsubsection{Replication in British Subsamples}

\subsection{Relationship Between Epistasis and Genetic Diversity}

\section{Discussion}\label{InterPath-Discussion}


\section{Online Methods}\label{InterPath-Online-Methods}

\subsection{UK BioBank Subsets}

\subsection{PLINK Analyses}

\subsection{MAPIT Analyses}

\subsection{GEMMA Analyses}

\subsection{InterPath Model}

\subsection{InterPath Analyses}

\section{URLs}\label{InterPath-URLs}

\section{Acknowledgments}\label{InterPath-Acknowledgments}

\section{Author Contributions}\label{InterPath-Author-Contributions}

\section{Competing Interests}\label{InterPath-Competing-Interests}

\begingroup
\bibliographystyle{apalike}
\setstretch{1.0}
%\bibliography{Main}
\endgroup

\clearpage
\section{Supplementary Material}

\captionsetup[figure]{name=Supplementary Figure}
\captionsetup[table]{name=Supplementary Table}
%\setcounter{figure}{0}


\begingroup
\bibliographystyle{apalike}
\setstretch{1.0}
%\bibliography{Supplementary}
\endgroup


\iffalse

\fi

\end{document}
