\documentclass[12pt, a4paper]{article}
\usepackage[english]{babel}
\usepackage[utf8x]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[a4paper]{geometry}
\usepackage{amsmath}
\usepackage{graphicx}
\usepackage[colorlinks=true, allcolors=blue]{hyperref}
\usepackage{epsfig,amsfonts}
\usepackage{natbib}
\usepackage{amssymb}
\usepackage{amsthm}
\usepackage{authblk}
\usepackage{setspace}
\usepackage{hypcap}
\usepackage{xr}
%From: https://tex.stackexchange.com/questions/337/how-to-change-certain-pages-into-landscape-portrait-mode
\usepackage{lscape}

\title{Supplementary: Differential complex trait architecture across humans: epistasis identified in non-European populations at multiple genomic scales}
\author[1,2]{Michael C. Turchin}
\author[1,3]{Isabella Ting}
\author[1,4,5,*]{Lorin Crawford}
\author[1,2,*,$\dag$]{Sohini Ramachandran}
\affil[1]{Center for Computational Molecular Biology, Brown University}
\affil[2]{Department of Ecology and Evolutionary Biology, Brown University}
\affil[3]{Department of Computer Science, Brown University}
\affil[4]{Department of Biostatistics, Brown University}
\affil[5]{Center for Statistical Science, Brown University}
\affil[$\ast$]{indicates these authors contributed equally}
\affil[$^\dag$]{To whom correspondence should be addressed: sramachandran@brown.edu}

\begin{document}

\maketitle

\section{Supplementary Note}\label{Supplementary-Note}

\subsection{Population Subset Quality Control}

We then conducted standard quality control (QC) procedures on each of these population subsets. Note that we focused our analyzes on the genotyped chip data throughout the project. First we conducted SNP-level QC by dropping variants that did not meet the following criteria:  minor allele frequency (MAF) >= .01, genotype missingness <= 5\%, and Hardy-Weinberg equilibrium test p-value >= $1\times10^{-6}$. We then conducted individual-level QC via the following steps. Individuals were removed if they did not have genotype missingness >= 5\%. Individuals were also removed if they were a 3\textsuperscript{rd} degree relative or more to someone else in the dataset; specifically the KING relatedness values provided with the UKB data were used to identify related individuals, and one individual from every pair of 3\textsuperscript{rd} degree or more relatives was removed. Individuals were also dropped if they were tagged by any of the following three flags from the UKB data: `het.missing.outliers', `putative.sex.chromosome.aneuploidy', and `excess.relatives'. Lastly, individuals were removed if they were determined to be PCA outliers; this was conducted by running FlashPCA (version 2.1) \citep{Abraham2017} in R on each population subset separately and identifying individuals that had PC values greater than 7 standard deviations away from the mean for any of the top 6 PCs. 

After this first round of QC procedures, we then proceeded to impute our current population subsets. Since most of the analyses in this project utilized genetic relatedness matrices (GRMs), and variants need to have no missing data for these GRMs, we used imputation primarily to maximize the number of genotyped SNPs that would not be dropped by this stringent threshold (as opposed to using imputation to increase the number of SNPs we were analyzing). To conduct this imputation, we uploaded our population subsets to the University of Michigan Imputation Server \citep{Das2016} and used the following options: Minimac3 for the imputation software, 1000G phase 3 v5 for the reference panel, and Eagle v2.3 for the phasing software. Completed imputed files were then downloaded from the Imputation Server afterwards and treated to further QC steps: imputed variants were intersected back to the original set of genotyped chip variants, variants with imputation quality scores < .3 were removed, and variants that had genotype missingness rates > 0\% were also removed. These steps represent the last of our QC and imputation procedures, and information on the final forms of our UKB population subsets can be found in Supplementary Table (table).

\clearpage


\section{Supplementary Figures}\label{Supplementary-Figures}


\begin{figure}[htbp]
\centering
\includegraphics[scale=.35]{Images/Supp/InterPath_Supp_Figure_MAPIT_vs3_BMI.png}
\caption[TBD]{\textbf{MAPIT Results QQ-Plots}. Shown are QQ-plots of our results from running MAPIT on our four initial UKB population subsets in height and BMI. On the $x$-axis are the -$\log_{10}$ of our expected $p$-values and the on the $y$-axis are on the -$\log_{10}$ of our observed $p$-values. We observe weak signals of marginal epistasis across most of the population subsets in height and only a few potential weak signals in BMI. We do observe potential differences across population subsets. However, this lack of signal may be a product of our limited sample sizes, emphasizing the need to increase power through other means, such as by methodological improvements.}
\label{InterPath-Supp-Figure-MAPIT-BMI}
\end{figure}
\clearpage

\begin{figure}[htbp]
\centering
\includegraphics[scale=.35]{Images/Supp/InterPath_Supp_Figure_GWAS_vs2_BMI.png}
\caption[TBD]{\textbf{GWAS Results QQ-Plots}.}
\label{InterPath-Supp-Figure-GWAS-BMI}
\end{figure}
\clearpage

\begin{figure}[htbp]
\centering
\hspace*{-.5cm}
\includegraphics[scale=.3]{Images/Supp/InterPath_Supp_Figure_MAPITvsGWAS_vs2.png}
\caption[TBD]{\textbf{MAPIT vs. GWAS Results}.}
\label{InterPath-Supp-Figure-MAPITvsGWAS}
\end{figure}
\clearpage

\begin{figure}[htbp]
\centering
\includegraphics[scale=.35]{Images/Supp/InterPath_Supp_Figure_Barplots_vs1.png}
\caption[TBD]{\textbf{Number of Significant Pathways from Running MAPIT-R on UKB subsets}. Shown are the number of significant pathways found from running MAPIT-R on each of our expanded collection of UKB population subsets using height and BMI in the KEGG database (REACTOME results can be found in Supplementary Figure XX). A pathway is considered significant if its MAPIT-R $p$-value is below XXX (.05 / XXX, the number of KEGG pathways tested). We find that the African population far and away has the largest number of significant pathways, both for each phenotype and for each pathway database. But in general we also tend to see significant pathways moreso in non-European populations than European populations.}
\label{InterPath-Supp-Figure-Barplots-All}
\end{figure}
\clearpage

\begin{figure}[htbp]
\centering
\includegraphics[scale=.35]{Images/Supp/InterPath_Supp_Figure_perm1_QQPlots_KEGG_vs1.png}
\caption[TBD]{\textbf{MAPIT-R Phenotype Permutation QQ-Plots}. Shown are QQ-plots for running MAPIT-R using a single, permuted version of the phenotypes for both Height and BMI with the KEGG pathways. Phenotypes were permuted within each population subset. Shown on the $x$-axis are the -$\log_{10}$ of our expected $p$-values and the on the $y$-axis are on the -$\log_{10}$ of our observed $p$-values. The dotted red line is our Bonferroni-corrected $p$-value threshold (.05 / XX KEGG pathways). We find across all population subsets that MAPIT-R displays the null behavior as expected when using permuted phenotypes.}
\label{InterPath-Supp-Figure-perm1-QQPlots-KEGG}
\end{figure}
\clearpage

\begin{figure}[htbp]
\centering
\includegraphics[scale=.35]{Images/Supp/InterPath_Supp_Figure_perm1_QQPlots_REACTOME_vs1.png}
\caption[TBD]{\textbf{TBD}. \\ Same as above (will be put in Supplementary).}
\label{InterPath-Supp-Figure-perm1-QQPlots-REACTOME}
\end{figure}
\clearpage

\begin{figure}[htbp]
\centering
\includegraphics[scale=.15]{Images/Supp/InterPath_Supp_Figure_pValHists_vs2.png}
\caption[TBD]{\textbf{TBD}. \\ Placeholder.}
\label{InterPath-Supp-Figure-10perms-pValHists}
\end{figure}
\clearpage

\begin{figure}[htbp]
\centering
\includegraphics[scale=1.5]{Images/Supp/InterPath_Supp_Figure_FDRs_AllPops_vs1.png}
\caption[TBD]{\textbf{TBD}. }
\label{InterPath-Supp-Figure-FDR-AllPops}
\end{figure}
\clearpage

\begin{figure}[htbp]
\centering
\includegraphics[scale=.15]{Images/Supp/InterPath_Supp_Figure_pValsVsNumSNPs_vs1.png}
\caption[TBD]{\textbf{MAPIT-R $p$-values Versus Number of SNPs in a Pathway}. \\ Shown is the relationship between MAPIT-R $p$-values and the number of SNPs found in a tested pathway across all populations in height and BMI within KEGG. Shown on the $x$-axis is the number of SNPs that were included for a given pathway and on the $y$-axis is shown that pathway's MAPIT-R -$\log_{10}$ $p$-value. As expected, we find a generally positive relationship between these two variables -- in other words, the more SNPs that are included the more likely it is that MAPIT-R will have a more significant $p$-value. We also find, however, that it is not simply the SNPs counts that are driving this signal; looking at this same comparison within the permuted phenotype analyses we find barely any positive relationship between SNP counts per pathway and a pathway's MAPIT-R -$\log_{10}$ $p$-value (Supplementary Figure \ref{InterPath-Supp-Figure-pValsVsNumSNPs-perm1}}
\label{InterPath-Supp-Figure-pValsVsNumSNPs}
\end{figure}
\clearpage

%\begin{figure}[htbp]
%\centering
%\includegraphics[scale=.15]{Images/Supp/InterPath_Supp_Figure_pValsVsNumSNPs_perm1_vs1.png}
%\caption[TBD]{\textbf{MAPIT-R $p$-values Versus Number of SNPs in a Pathway, Permuted Phenotypes}. \\ Shown is the relationship between MAPIT-R $p$-values and the number of SNPs found in a tested pathway across all populations in permuted height and BMI within KEGG. Shown on the $x$-axis is the number of SNPs that were included for a given pathway and on the $y$-axis is shown that pathway's MAPIT-R -$\log_{10}$ $p$-value. Unlike with the observed data (Figure \ref{InterPath-Supp-Figure-pValsVsNumSNPs}), we do not see a relationship between the number of SNPs included in a pathway and that pathway's -$\log_{10}$ MAPIT-R $p$-value. This shows that while having more SNPs allows MAPIT-R to generate more power to detect epistatic interactions, our significant results are not due solely to having more SNPs.}
%\label{InterPath-Supp-Figure-pValsVsNumSNPs-perm1}
%\end{figure}
%\clearpage

\begin{figure}[htbp]
\centering
\includegraphics[scale=.225]{Images/Supp/InterPath_Supp_Figure_Heatplots_REACTOME_vs1.png}
\caption[TBD]{\textbf{TBD}. \\ See above (this would be a supplementary figure).}
\label{InterPath-Supp-Figure-Heatplots-REACTOME}
\end{figure}
\clearpage

%\begin{figure}[htbp]
%\centering
%\includegraphics[scale=.15]{Images/Main/InterPath_Main_PopCompDotPlots_African_vs1.png}
%\caption[]{\textbf{MAPIT-R Cross-Population KEGG $p$-value Comparisons: African}. Shown are comparisons of each population subset's MAPIT-R $p$-value for Height and BMI in KEGG against the African subset's MAPIT-R $p$-values. On the $x$-axis are the African subset's observed -$\log_{10}$ $p$-value and on the $y$-axis is the other population's observed -$\log_{10}$ $p$-value. Dotted red lines represent Bonferroni-corrected $p$-value thresholds for each population/phenotype combination.}
%\label{InterPath-Main-Figure-PopCompDotPlots}
%\end{figure}
%\clearpage

\begin{landscape}
\begin{figure}[htbp]
\centering
\hspace*{-.5cm}
\includegraphics[scale=1]{Images/Supp/InterPath_Supp_Figure_BritReps_Barplot_Temp.png}
\caption[TBD]{\textbf{TBD}. }
\label{InterPath-Supp-Figure-BritReps-Barplots}
\end{figure}
\end{landscape}
\clearpage

%\begin{figure}[htbp]
%\centering
%\includegraphics[scale=.15]{Images/Supp/InterPath_Supp_Figure_PhenoCompDotPlots_vs1.png}
%\caption[TBD]{\textbf{MAPIT-R Phenotype $p$-value Comparisons}. \\ Shown is a comparison of the MAPIT-R $p$-values for both phenotypes analyzed across all population subsets in the KEGG database. Shown on the $x$-axis is the observed height -$\log_{10}$ MAPIT-R $p$-value and shown on the $y$-axis is the observed BMI -$\log_{10}$ MAPIT-R $p$-value. Dotted red lines represent Bonferroni-corrected $p$-value thresholds for each population/phenotype combination (.05 / number of KEGG pathways analyzed). In general we observe a correlation between MAPIT-R $p$-values between each phenotype. In the African subset, where we have the most observed power, we find pathways that are significant in both phenotypes as well as each phenotype separately. Additionally, we observe, among marginally significant pathways, stronger MAPIT-R signals in BMI than height -- this is in line with previous observations that BMI may contain higher levels of epistasis than height (citations).}
%\label{InterPath-Supp-Figure-PhenoCompDotPlots}
%\end{figure}
%\clearpage

\begin{figure}[htbp]
\centering
\includegraphics[scale=.15]{Images/Supp/InterPath_Supp_Figure_IBS_AllPops_vs2.png}
\caption[TBD]{\textbf{IBS Proportions vs. MAPIT-R $p$-values in All UKB Subsets}}
\label{InterPath-Supp-Figure-IBS-AllPops}
\end{figure}
\clearpage

\begin{figure}[htbp]
\centering
\includegraphics[scale=.35]{Images/Supp/InterPath_Supp_Figure_IBS_AllPopComps_vs2.png}
\caption[TBD]{\textbf{IBS in Top Pathways Across Populations}.}
\label{InterPath-Supp-Figure-IBS-AllPopComps}
\end{figure}
\clearpage

\section{Supplementary Tables}\label{Supplementary-Tables}

\begin{table}[ht]
\centering
\begin{tabular}{ccccc}
  \hline
\textbf{UK BioBank} & \textbf{Individuals} & \textbf{SNPs} & \textbf{KEGG} & \textbf{REACTOME} \\
\textbf{Subset} & & & \textbf{Pathways} & \textbf{Pathways}  \\
  \hline
\textbf{Original Subsets:} & & & & \\
African & 3111 & 374466 & 180 & 658 \\ 
British.Ran4000 & 3848 & 600006 & 173 & 650 \\ 
British.Ran10000 & 9603 & 597298 & 186 & 669 \\ 
Caribbean & 3833 & 410017 & 181 & 661 \\ 
Chinese & 1448 & 345221 & 153 & 626 \\ 
Indian & 5077 & 505854 & 181 & 662 \\ 
Irish & 11575 & 588324 & 186 & 671 \\ 
Pakistani & 1581 & 516806 & 141 & 596 \\ 
\\
\textbf{British Replicates:} & & & & \\
British.Ran4000.2 & 3869 & 599381 & 173 & 650 \\ 
British.Ran4000.3 & 3836 & 600654 & 173 & 649 \\ 
British.Ran4000.4 & 3838 & 599829 & 173 & 650 \\ 
British.Ran4000.5 & 3853 & 599442 & 173 & 650 \\ 
British.Ran10000.2 & 9628 & 597577 & 186 & 669 \\ 
British.Ran10000.3 & 9636 & 597486 & 186 & 669 \\ 
British.Ran10000.4 & 9593 & 597369 & 186 & 669 \\ 
British.Ran10000.5 & 9596 & 597507 & 186 & 669 \\ 
  \hline
\end{tabular}
\caption[TBD]{\textbf{UK BioBank Subset Statistics}. \\ }
\label{InterPath-Supp-Table-UKBPopStats}
\end{table}
\clearpage

\begin{landscape}
\begin{figure}[htbp]
\centering
\hspace*{-2.5cm}
\includegraphics[scale=1]{Images/Supp/InterPath_Supp_Table_TopPathwayGeneCounts.png}
\caption[TBD]{\textbf{Table of Most Present Genes in Significant MAPIT-R Pathways} \textcolor{red}{this will be a table}}
\label{InterPath-Supp-Tables-GeneCounts}
\end{figure}
\end{landscape}
\clearpage

\begin{figure}[htbp]
\centering
\includegraphics[scale=1.5]{Images/Supp/InterPath_Supp_Figure_FDRs_BritReps_vs1.png}
\caption[TBD]{\textbf{TBD}. }
\label{InterPath-Supp-Figure-BritReps-FDRs}
\end{figure}
\clearpage


\begingroup
\bibliographystyle{apalike}
\setstretch{1.0}
\bibliography{Main}
\endgroup


\iffalse

SCRAP

Additionally, it is unlikely that, even if such an ascertainment bias existed, it could continually be directionality consistent across pathways that span the genome. If this were an approach that aggregated single SNP tests, then directionality would not be an issue and one might become concerned about continually incorporating a persistent signal from ascertainment bias; however, because this is a joint test on multiple SNPs as once, it is far less likely that differences between genotype chip SNP distributions and non-European population SNP distributions tracts in exactly the same direction across the entire genome.

\subsubsection{Pairwise SNP Epistasis}

Our first approach to investigate SNP-level epistasis was to run the canonical pair-wise interaction model as employed by PLINK's \texttt{--epistasis} method \citep{Purcell2007} (see Online Methods and Supplementary Note for details). Here, we directly estimated the effect size of every possible SNP pairwise interaction and calculated whether it was statistically significant or not.  

Running this method on each of our four UKB population subsets, we chose to look at the 'proportion of marginally significant tests' per SNP, where marginal significance is defined as any test with a $p$-value < $1\times10^{-4}$ (Figure \ref{InterPath-Supp-Figure-PLINK}). We chose to look at proportion of tests due the varying number of SNPs across each population subset. What we find is that indeed there is variation between human ancestries in the amount of pairwise SNP epistasis across the genome. We also find that both the African and Caribbean population subsets show greater proportions of marginal interactions than the British population subset despite both non-European populations having lower sample sizes. 

\begin{figure}[t]
\centering
\includegraphics[scale=.35]{Images/Supp/InterPath_Supp_Figure_PLINK_vs2.png}
\caption[PLINK SNP Pairwise Epistasis]{\textbf{PLINK SNP Pairwise Epistasis.} \\ Shown are results from running PLINK's SNP-by-SNP pairwise epistasis method, \texttt{--epistasis}, on four of our initial UKB human population subsets in height and BMI. Specifically the $y$-axis shows the proportion of marginally significant interactions found per-SNP, where marginal significance is defined as the epistastic test returning a $p$-value $<$ $1\times10^{-4}$, and the $x$-axis shows the top 50 SNPs for this metric in each of the population subsets. The African subset contained 374,466 total SNPs, the British Random 4k subset contained 600,006 total SNPs, the Caribbean subset contained 410,017 total SNPs, and the Indian subset contained 505,854 total SNPs. We observe across each population different levels of marginal pairwise SNP significance as well as different patterns of significance. Additionally, both the African and Caribbean subsets contain the strongest signals.}
\label{InterPath-Supp-Figure-PLINK}
\end{figure}

\subsection{Genome Level Epistasis}\label{InterPath-Results-GenomeEpistasis}

To investigate epistasis in multiple human populations, we first extracted a variety of human ancestries from the UKB (see Online Methods). We collected a total of 8 UKB subsets with an average sample size of XXXX among the non-European populations (Supplementary Table 1), including African, Indian, Chinese, and Pakistani subsets. To maximize our sample sizes per subset, we focused on height and body mass index (BMI) as our complex traits of interest. 

The first approach we took to investigate epistasis was to look at genome-wide estimates of phenotypic PVE. Specifically, we setup a linear mixed model that incorporated both additive and epistatic interactions, and fit each of these variance components using GEMMA \citep{Zhou2012} (see Online Methods and Supplementary Note for details). Calculating these variance component PVEs for height and BMI in each of our populations, we indeed find differences across each human ancestry (Table 1).

\begin{figure}[ht]
\centering
\includegraphics[scale=1]{Images/Table1_Placeholder.png}
\caption[TBD]{\textbf{TBD}. \\ this will be a table not a figure.}
\label{IntrePath-Main-Table-GEMMA}
\end{figure}

In height, we do not see anything immediately new. For the additive effects $G_1$ we mainly calculate PVEs across each population within our range of expectation (between XX and XX (citation)). Additionally, we do not currently see much evidence for second-order effects $G_2$. However, looking at BMI we see a different result. Across all populations we see non-trivial PVEs for $G_2$, ranging from XX to XX. We also see larger values in multiple non-European populations. In particular we see a particularly strong result in XXX. This might be beginning evidence that indeed, non-European populations have more potential for picking up signals of higher-order interactions in the genome.   

\fi

\end{document}
